cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/DefaultStore.java

```diff
@@ -930,6 +940,12 @@ private AppStateTable getAppStateTable(StructuredTableContext context) throws Ta
     return new AppStateTable(context);
   }
 
+  private void emitMetrics(String namespace, String appName, String metricName) {
+    Map<String, String> tags = ImmutableMap.of(io.cdap.cdap.common.conf.Constants.Metrics.Tag.NAMESPACE, namespace,
```

Comment:
I believe we need more context tags, as this should be part of appfabric service context?

Also, why we need the fully qualified name? Can we import the class?